docs/permissions-and-roles.md

```diff
@@ -0,0 +1,44 @@
+# Permissions and Roles
+This document will detail the permissions and roles required for accessing a subset of endpoints within the Authoring Acceptance Gateway (AAG) and interacting with user-created
+ `CriteriaItem`s.
+
```

Comment:
This document has come from the realisation that there is word of mouth knowledge on how the permissions work on the service. I've tried writing the document to be simple and maintainable but hopefully insightful in how AAG works.